packages/libluajit/build.sh

```diff
@@ -2,79 +2,52 @@ TERMUX_PKG_HOMEPAGE=https://luajit.org/
 TERMUX_PKG_DESCRIPTION="Just-In-Time Compiler for Lua"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1:2.1.1763148144"
+TERMUX_PKG_VERSION="1:2.1.1763148144+g5c8cee3"
```

Comment:
The commit hash is now added by Arch and provides us with both an easier method of determining the commit and a secondary layer of validation that we have the correct commit and timestamp pair.